src/ansible_navigator/configuration_subsystem/navigator_post_processor.py

```diff
@@ -655,6 +656,24 @@ def lintables(
                         prefix=ExitPrefix.HINT,
                     ),
                 )
+            else:
+                try:
+                    subprocess.run("ansible-lint --version", shell=True, check=True)
+                except subprocess.CalledProcessError:
+                    exit_messages.append(
+                        ExitMessage(
+                            message=("ansible-lint does not seem to be installed correctly.")
+                        ),
+                    )
+                    exit_messages.append(
+                        ExitMessage(
+                            message=(
+                                "Ensure the command `ansible-lint --version` can be run prior to"
```

Comment:
Can we make this message more generic? Something like, `Ensure that ansible-lint is installed in the environment`?

Reply:
I prefer the original, this check is follow on to finding the executable, to ensure not only that is it installed, but it is a working installation.

Reply:
Okay. @shatakshiii, I think you have to revert the changes :)

Reply:
reverted that change :)